packages/flutter/test/widgets/editable_text_test.dart

```diff
@@ -193,6 +193,80 @@ void main() {
     skip: kIsWeb, // [intended]
   );
 
+  // Regression test for https://github.com/flutter/flutter/issues/127597
+  testWidgets(
+    'TapRegionSurface detects outside right click',
```

Comment:
The test doesn't have anything to do with right click? It is for clicks outside of the TextFieldTapRegion, not specifically _right_ clicks.

Reply:
Yes, I will revise the description